### DISCUSSION TITLE
PCM4 Graphic problem - missing AM/FM/DAB on source selection

### DISCUSSION THREAD

**Discussion by jacobello87** [Mr-MIBonk/M.I.B._More-Incredible-Bash#259] [Q&A]
Hello, after US->EU conversion via MIB and update to MHI2_ER_POG11_K5126_1 FW i am facing some graphic problem on PCM4 unit. 
maybe some help? :-) thank you

![IMG_3567](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/54706361/f69c19b8-40c5-4a27-b46e-0f2425e4855f)

**Comment by harman-f** [comment]
What are you missing?

**Comment by jacobello87** [comment]
It should look like thisOn 22 Jun 2023, at 07:08, harman-f ***@***.***> wrote:﻿
What are you missing?

**Reply by harman-f** [reply]
the picture did not make it ;)

**Reply by jacobello87** [reply]
![image0](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/54706361/9af249ed-861c-4ba9-bdec-23aa1252fee0)
